base_nuclei_scan.yml

```diff
@@ -0,0 +1,11 @@
+# https://github.com/aquasecurity/trivy#gitlab-ci
+
+.base_nuclei_scan:
+  stage: .post
+  image: projectdiscovery/nuclei:latest
+  allow_failure: true
+  environment:
+    name: ${CI_COMMIT_REF_SLUG}-dev2
+    url: https://${CI_ENVIRONMENT_SLUG}-${CI_PROJECT_NAME}.${KUBE_INGRESS_BASE_DOMAIN}
```

Comment:
This looks like our socialgouv/autodevops specific.
We should move it to our autodevops file and allow the use of `${AUTO_DEVOPS_DEV_ENVIRONMENT_NAME}`